src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -476,7 +477,7 @@ public override ImmutableArray<ImmutableArray<TypeWithAnnotations>> GetTypeParam
                 var syntax = Syntax;
                 var diagnostics = BindingDiagnosticBag.GetInstance(_declarationDiagnostics);
                 var constraints = this.MakeTypeParameterConstraintTypes(
-                    _binder,
+                    ParameterBinder,
```

Comment:
>ParameterBinder [](http://example.com/codeflow?start=20&length=15)

Consider renaming corresponding parameter of ```MakeTypeParameterConstraintTypes``` to clearly reflect what kind of binder it expects.